frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -0,0 +1,314 @@
+                          type="button"
+                          aria-label="close"
+                          onClick={() => {
+                            setErrorMessageText("");
+                            setErrors([]);
+                          }}
+                        >
+                          <FontAwesomeIcon icon={faXmark} />
+                        </button>
+                        <p className="usa-alert__text">{errorMessageText}</p>
+                      </div>
+                    </div>
+                    {errors.length > 0 && (
+                      <table className="usa-table usa-table--borderless">
+                        <thead>
+                          <tr>
+                            <th
+                              className={"thick-bottom-border padding-left-0"}
+                            >
+                              Edits needed
+                            </th>
+                            <th
+                              className={"thick-bottom-border padding-left-0"}
+                            >
+                              Areas requiring edits
+                            </th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                          {errors.map((e, i) => {
+                            return (
+                              <tr key={"error_" + i}>
+                                <td className={"no-bottom-border"}>
+                                  {e?.["message"]}{" "}
+                                </td>
+                                <td className={"no-bottom-border"}>
+                                  Row(s): {e?.["indices"]}
+                                </td>
+                              </tr>
+                            );
+                          })}
+                        </tbody>
+                      </table>
+                    )}
+                  </div>
+                )}
+                <FormGroup className="margin-bottom-3">
+                  {status === "submitting" ? (
+                    <div className={"usa-file-input"}>
+                      <div className={"usa-file-input__target"}>
+                        <div className={"margin-top-1"}>
+                          <span className="usa-file-input__drag-text font-sans-xs text-bold">
+                            Uploading patient information...
+                          </span>
+                          <div className={"margin-top-1"}>
+                            <img
+                              src={iconLoader}
+                              alt="submitting"
+                              className={"square-5"}
+                            />
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  ) : (
+                    <FileInput
```

Comment:
I'm seeing a difference in styling between Figma and on your PR -- wanted to confirm if it was intentional:

PR
<img width="780" alt="Screen Shot 2022-11-01 at 18 17 40" src="https://user-images.githubusercontent.com/20211771/199360090-5a23ce59-8e49-4c14-93be-2dcd851d7c48.png">


Figma
<img width="349" alt="Screen Shot 2022-11-01 at 18 15 08" src="https://user-images.githubusercontent.com/20211771/199360110-b434b7a8-823b-4645-a5e1-361433535254.png">

Reply:
I'm working with kenny for how the file should react, but if I make it the bigger size then the file takes up the whole box.

Reply:
Kenny was 🆗 to keep the box that small size!